### DISCUSSION TITLE
What's the best way to edit mdx then return the edited mdx as an output?

### DISCUSSION THREAD

**Discussion by hahnbeelee** [mdx-js/mdx#2160] [Q&A] [answered]
I want to make a remark plugin to edit my AST of mdx content, then I want the edited MDX as output. What's the best way to go about doing this?

For most of the processes that edit the AST, the output has been runnable javascript, but I want MDX as the output.

**Comment by wooorm** [comment, marked as answer]
Use unified with remark and `remark-mdx`, it enables the MDX syntax in markdown.

**Reply by hahnbeelee** [reply]
Thank you! This worked :)
